platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -219,7 +241,11 @@ function UpgradeASICFW() {
 function UpgradeCPLDFW_Worker() {
     local -r _CPLD_BURN_FILE="${1}"
     local -r _CPLD_REFRESH_FILE="${2}"
-    local -r _ASIC_DEV="$(find /dev/mst -iname '*_pciconf0')"
+    local -r _ASIC_DEV="$(find /dev/mst -iname '*_pci_cr0')"
```

Comment:
>_pci_cr0 [](start = 49, length = 8)

Could you explain the motivation of this change? Is it a bug fix?

Reply:
@qiluo-msft done - updated the description